NOM_visualizations/python/nom_data.ipynb

```diff
@@ -0,0 +1,3353 @@
+{
```

Comment:
<p>Again, consider mentioning why you are grabbing the <code>has_input</code>  as well.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microbiomedata/nmdc_notebooks/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>